taskcluster/docker/skopeo/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM golang:1.14 as skopeo
+
+WORKDIR /go/src/
+RUN ["/usr/bin/git", "clone", "--no-checkout", "--depth=1", "--branch=v1.1.0", "https://github.com/containers/skopeo", "."]
+RUN ["/usr/bin/git", "checkout", "63085f5bef1131aa9ec0907a5c8d66b67de7c4b2"]
+ENV GO111MODULE=on CGO_ENABLED=0
```

Comment:
I don't like all these external dependencies and odd-looking commands...but I don't have a better suggestion.

Reply:
Ack.  I should have mentioned a lot of this was stolen from https://github.com/mozilla-releng/shipit/pull/336 so that's arguably a pre-existing problem.